app/views/admin/resources/_update_links.html.haml

```diff
@@ -0,0 +1,7 @@
+= link_to admin_conference_resource_path(conference.short_title, resource, increment_used_resource_flag: 1), method: :put, title: "Use 1 #{resource.name}", class: "btn btn-primary action-btn #{ 'disabled' if resource.used== resource.quantity }", remote: true do
```

Comment:
I think it makes more sense that I give out something (which is my default action as an info desk volunteer) and I click +
(which should increase `used`)